db/migrate/20200529160454_create_copies.rb

```diff
@@ -0,0 +1,16 @@
+class CreateCopies < ActiveRecord::Migration[6.0]
+  def self.up
+    create_table :copies do |t|
+      t.belongs_to :book
+      t.belongs_to :library_location
```

Comment:
Let's change these to be not null and add a foreign key, which is good practice.
```suggestion
      t.belongs_to :book, null: false, foreign_key: true
      t.belongs_to :library_location, null: false, foreign_key: true
```